chrome/extension/background/keyBindings.js

```diff
@@ -0,0 +1,29 @@
+import Immutable from 'immutable';
+import { callWithStore } from '../../../app/store/utils';
+import { rememberArticle } from '../../../app/actions/article';
+
+callWithStore((store) => {
```

Comment:
While giving a name to the method I was concentrated on the `callback(store)` part, but you are right method also sets up redux, reads from localStorage...